### DISCUSSION TITLE
Pattern matching tab is throwing an error

### DISCUSSION THREAD

**Discussion by myshenoy** [SwiftFiddle/swiftregex#202] [General]
Pattern match tab is throwing an error:
```
swift-backtrace: error while loading shared libraries: libswift_Backtraci
```

Attaching screenshot for reference
<img width="1477" alt="Screenshot 2023-08-10 at 3 53 58 PM" src="https://github.com/SwiftFiddle/swiftregex/assets/103593906/45f0f18a-1c1c-4705-8a1c-f345dde5d19d">

**Comment by kishikawakatsumi** [comment]
Thanks for reporting. Fixed by https://github.com/SwiftFiddle/swiftregex/pull/203
